File: Homework/fizzbuzz.pl
#!/usr/bin/perl

#FizzBuzz Program

print "Welcome to FizzBuzz Program \n\n";
print "Enter a Number: ";
chomp($num = <STDIN>);

if ($num % 3 == 0 && $num % 5 == 0){
	print "FIZZBUZZ \n";
}
	elsif ($num % 3 == 0){
	print "FIZZ \n";
}
	elsif ($num % 5 == 0){
	print "BUZZ \n";
}

File: Homework/fizzbuzz.py
#!/usr/bin/py

print('Welcome to FIZZBUZZ Python')

a =input('Enter a Number: ')

if a % 3 == 0 | a % 5 == 0:
	print ('FIZZBUZZ')
elif a % 3 == 0:
	print('FIZZ')
elif a % 5 == 0:
	print ('BUZZ')

